src/opencl-platform/cl_init_platform.cpp

```diff
@@ -291,8 +291,13 @@ bool cl_check_init(void) {
             /* select device based on cli arguments */
             string tmpAttrData = attr_data;
             
+            if((dev == 0) && 
+                (platf == 0)) {
+                device = device_list[dev];
+                cout << "<----- SELECTED";
+            }
             // always select last device of type GPU
-            device = device_list[dev];
+            // device = device_list[dev];
```

Comment:
What's the logic behind removing this?

Reply:
I added an If that contains that line.
The reason for this change is that I wanted to see the selected device (see below output).
OpenCL platform query & init...
OpenCL init devices query type: ALL
Platforms found: 2
Platform 0 NVIDIA Corporation OpenCL 1.2 CUDA 10.2.185
	Devices found 1
	Device 0 GeForce GTX 1650 with Max-Q Design <----- SELECTED
Platform 1 The pocl project OpenCL 1.2 pocl 1.4, None+Asserts, LLVM 9.0.1, RELOC, SLEEF, DISTRO, POCL_DEBUG
	Devices found 1
	Device 0 pthread-Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz